2016-10-14-TWIT.md

```diff
@@ -0,0 +1,32 @@
+# This Week in Tessel: Hacktoberfest, Rust update, and conferences, oh my!
+
+Hello, Tesselators!
+
+**TL;DR:**
+* [Hacktoberfest](https://hacktoberfest.digitalocean.com) is back, and we have [open issues](https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3AHacktoberFest+user%3Atessel+) labeled for participants!
+* We just released an update to [t2-cli](https://github.com/tessel/t2-cli/releases/tag/v0.1.0), enabling local compilation for Rust programs!
```

Comment:
Make "Rust programs" link to [tessel-rust](http://github.com/tessel/tessel-rust)